### ISSUE TITLE
Display notification to the user to be patient upon switching sections.

### ISSUE THREAD

**Issue by mgurley** [NUARIG/surveyor#268]
Display notification to the user to be patient upon switching sections.   Give the user some kind a visual clue to be patient.  Even though, surveyor has been dramatically speeded up, I have still witnessed users click the next section button repeatedly and ask 'Why is it doing nothing?'.

**Comment by yoon** [comment]
@mgurley Is this still applicable?

**Comment by mgurley** [comment]
Yes, this is still applicable.  We have a local solution in one of our projects that converts the section, next and previous  buttons into links and then adds a call to the blockui plugin upon click of these links.   It also adds a hidden button at the top of the page that returns false on click and instructs the main surveyor form to return false upon submit. This solution gives up on supporting clients without JavaScript enabled but does fix one of the causes of duplicate responses-- enabling the user to accidentally push the enter key without any negative consequences.  Also, it gives the user feedback to be patient upon the switching of sections -- which is still slow enough to cause users to click the next or previous button before the new page has been rendered.

**Comment by yoon** [comment]
@mgurley Has this been fixed by #328?
